core/src/main/java/nu/yona/server/subscriptions/entities/User.java

```diff
@@ -40,7 +40,8 @@
 	@Column(unique = true)
 	private String mobileNumber;
 
-	private LocalDateTime creationTime;
+	// YD-666 Change to LocalDate when all users have migrated their data
```

Comment:
Why can't it immediately be a `LocalDate`?

Reply:
Because we then cannot migrate the time part of it anymore in the migration step.

Reply:
OK, the comment is about the `creationTime` and not about the `roundedCreationDate`?
Why don't we make `roundedCreationDate` a `LocalDate` instead of `LocalDateTime`?

Reply:
It at least needs a bigger comment :)
This is how it works:
* This PR renames ``creationTime`` into ``roundedCreationDate``
* The migration step copies the date/time into the private (encrypted) data and rounds the time to 'start of day'
* If the user is active for more than 30 days, the date is set to the first of the month

After everyone has executed the migration step, the ``roundedCreationDate`` column won't contain any times (other than 00:00) anymore, so the data type can be changed without losing data.

Makes sense? If yes, I'll explain this in a comment in the source code.

Reply:
Makes sense, I missed that the existing column was renamed.